docs/guide/controllers.md

```diff
@@ -560,3 +560,62 @@ final class LatestContentController extends AbstractController
 }
 
 ```
+
+### Content query Field Type
+
+The [Content query Field Type](../api/field_type_reference.md#content-query-field-type)
+enables you to get query results in a Content item's Field.
+
+Use it by adding a Content query Field Type to your Content Type.
+
+Select a predefined [Query Type](#query-controller) from a list
+and provide the parameters that are required by the Query Type, e.g.:
+
+```
+parentLocationId: '@=mainLocation.id'
+```
+
+Select the Content Type of items you want to return in the **Returned type** dropdown.
+To take it into account, your Query Type must filter on the Content Type.
+You can then provide the selected Content Type through the `returnedType` variable:
+
+```
+contentType: '@=returnedType'
+```
+
+The following variables are available in parameter expressions:
+
+- `returnedType` - the identifier of the Content Type selected in the **Returned type** dropdown
+- `content` - the current Content item
+- `contentInfo` - the current Content item's ContentInfo
+- `mainLocation` - the current Content item's main Location
+
+#### Content query Field Type view
+
+Configure the Content query Field Type's view using the `content_query_field` view type:
+
+``` yaml
+content_view:
+    content_query_field:
+        blog_posts:
+            match:
+                Identifier\ContentType: blog
+                Identifier\FieldDefinition: posts
+            template: "blog_posts.html.twig"
+```
+
+Query results are provided to the template in the `items` variable:
+
+``` html+twig
+{% for item in items %}
+    {{ render(controller("ez_content:viewAction", {
+        "contentId": item.id,
+        "content": item,
+        "viewType": itemViewType
+    })) }}
+{% endfor %}
+```
+
+The default view type is `line`, defined under `itemViewType`.
+You can change it by passing a different view to `viewType` in the template.
+You can also define a different view type globally by overriding the default `parameters.ezcontentquery_item_view` parameter.
```

Comment:
For multiple reasons (e.g it forces us to keep BC promise on this parameter), I'm against documenting it at all. Is there is any other way to provide this configuration?

Besides that the parameter name is incorrect. 

```suggestion
You can also define a different view type globally by overriding the default `ezcontentquery_item_view` parameter.
```

But for multiple reasons (e.g it forces us to keep BC promise on this parameter), I'm against documenting it at all. Is there is any other way to provide this configuration?

Reply:
I don't disagree about not documenting it. The daily usage will be by providing a different view type from a custom template anyway.